dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageChecker.java

```diff
@@ -87,18 +89,15 @@ public static StaticLinkageChecker create(boolean onlyReachable,
 
   StaticLinkageChecker(
       boolean reportOnlyReachable, ClassDumper classDumper, Iterable<Path> entryPoints) {
-    this.reportOnlyReachable = reportOnlyReachable;
-    this.classDumper = classDumper;
-    this.entryPoints = ImmutableSet.copyOf(entryPoints);
-    this.paths = ArrayListMultimap.create();
+    this(reportOnlyReachable, classDumper, entryPoints, ArrayListMultimap.create());
   }
   
   StaticLinkageChecker(boolean reportOnlyReachable, ClassDumper classDumper,
       Iterable<Path> entryPoints, ListMultimap<Path, DependencyPath> paths) {
     this.reportOnlyReachable = reportOnlyReachable;
-    this.classDumper = classDumper;
-    this.entryPoints = ImmutableSet.copyOf(entryPoints);
-    this.paths = paths;
+    this.classDumper = Preconditions.checkNotNull(classDumper);
+    this.entryPoints = ImmutableSet.copyOf(Preconditions.checkNotNull(entryPoints));
```

Comment:
You don't need this one. `ImmutableSet.copyOf(null)` will throw.

Reply:
done